### PR TITLE
Function naming

### LINKED ISSUE #194: checkAuthroisedAPIIDentifier should be renamed
As spotted in #193 the Site Service function `checkAuthroisedAPIIDentifier()` should probably have it's typos removed and be renamed to `checkAuthorisedAPIIdentifier()`.

### PR DESCRIPTION
**Pull request by GRyall**
fixes #194 

depends on ~#193~

### CODE REVIEW

File: lib/Gocdb_Services/ServiceService.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?php
+checkAuthorisedAPIIdentifier<?php
```
Comment (jrha) [inline lib/Gocdb_Services/ServiceService.php]: Wait, what's this?

Comment (GRyall) [reply lib/Gocdb_Services/ServiceService.php]: 🤦‍♂️

Comment (GRyall) [reply lib/Gocdb_Services/ServiceService.php]: Fixed

Comment (jrha) [reply lib/Gocdb_Services/ServiceService.php]: Oh no it isn't!

Comment (jrha) [reply lib/Gocdb_Services/ServiceService.php]: :ghost:

Comment (GRyall) [reply lib/Gocdb_Services/ServiceService.php]: Oh yes it is! (now)

**Comment by GRyall** [comment]
Rebased at request of @gregcorbett to simplify review

**Pull request merged**